build/scripts/win/terrame-conf.bat

```diff
@@ -40,12 +40,12 @@ if "%_TERRALIB_MODULES_DIR%" == "" (
 
 :: Checking qt5 dir or setting default
 if "%_Qt5_DIR%" == "" (
-  set _Qt5_DIR=C:\Qt\5.11.2\msvc2017_64
+  set _Qt5_DIR=C:\Qt\5.10.1\msvc2017_64
```

Comment:
Are we now using an older qt version?

Reply:
This version is the version on CI server, anyway, jenkins set this variable, it doesn't go into this `if`. My local machine had this qt version.